README.md

```diff
@@ -99,7 +99,7 @@ All feedback and suggestions are welcome!
 ## Contributors
 
 This project exists thanks to all the people who contribute. [[Contribute]](CONTRIBUTING.md).
-<a href="https://github.com/parcel-bundler/parcel/graphs/contributors"><img src="https://opencollective.com/parcel/contributors.svg?width=890" title="contributors" alt="contributors" /></a>
```

Comment:
What's broken about this? It redirects to the contributors page of the github repo. Which is more up to date than the opencollective clone of the same list with less details.

Reply:
Sorry, the link is wrong in `CONTRIBUTING.md`. I will close this PR and open a new one.